source/extensions/filters/network/kafka/serialization.h

```diff
@@ -96,7 +97,7 @@ class Int8Deserializer : public IntDeserializer<int8_t> {
 public:
   int8_t get() const override {
     int8_t result;
-    memcpy(&result, buf_, sizeof(result));
+    memcpy(&result, buf_, sizeof(int8_t)); // NOLINT(safe-memcpy)
```

Comment:
safeMemcpy?

Reply:
Isn’t it better to do an assignment here? i.e. ```int8_t result = buf_[0];```

Reply:
If that works; I imagine there are possibly signedness issues the compiler objects to that motivated this.